bootcamp/tutorials/integration/llama_agents_metadata.ipynb

```diff
@@ -0,0 +1,1130 @@
+{
```

Comment:
<p>this is good to explain why using any particular model.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1401/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>